### ISSUE TITLE
LTA error message when passing a Sequence to a signature expecting a List

### ISSUE THREAD

**Issue by mustafaaydn** [rakudo/rakudo#4864]
## The Problem
Passing a Sequence to a function expecting a List gives a less than awesome run time error:

```
sub fun(List) {}
fun 2...8;
```
```
Internal error: inconsistent bind result
```

## Expected Behavior
Perhaps a compile time error as it happens when we pass an Int, or a more descriptive runtime error as it happens when we pass a Range.

## Environment
Windows 10 with
> Welcome to 𝐑𝐚𝐤𝐮𝐝𝐨™ v2021.07.
> Implementing the 𝐑𝐚𝐤𝐮™ programming language v6.d.
> Built on MoarVM version 2021.07.

**Comment by vrurg** [comment]
It is not about LTA but rather about a real bug. #4909 happens to be a simple fix.

**Comment by vrurg** [comment]
Fixed with #4909

**Comment by vrurg** [comment]
Closing via #4951
